src/sprites/entity.py

```diff
@@ -45,7 +45,6 @@ def __init__(
         self.tool_index = self.current_tool.value - FarmingTool.get_first_tool_id().value
 
         self.tool_active = False
-        self.just_used_tool = False
```

Comment:
You should not remove this variable, it is used to ensure the tool isn’t reset ad infinitum when you press a key

Reply:
I’m willing to put this variable back, but what it will effectively do is, at frame -1, it uses the tool and sets just_used_tool to true, and at the very next frame, it sets it back to false, so it wouldn’t serve any purpose.

Right now, we can only use the tool once because the keyboard checks for key presses, not if the key is held down, but for the mouse, it uses it continuously, yes.

Even before I refactored the animation, it used the tool continuously with the mouse, and the just_used_tool variable was useless too.

Reply:
Hmm

Reply:
I would check this with my computer, but I’m away from home right now